src/Sentry.Extensions.Logging/SentryLogger.cs

```diff
@@ -78,6 +78,11 @@ public bool IsEnabled(LogLevel logLevel)
                             };
                             break;
```

Comment:
Won't this line prevent the code below from running?

Reply:
That's true. Made sense before we added the new block but now if the `{OriginalFormat}` comes in first, the rest won't be added.
I'll fix it before publishing.